domain/ticket-domain/src/main/java/com/uket/domain/ticket/service/TicketService.java

```diff
@@ -51,4 +52,13 @@ public List<Ticket> findAllTicketsByUserId(Long userId) {
         return ticketRepository.findAllByUserId(userId);
     }
 
+    public void cancelTicketByUserIdAndId(Long userId, Long ticketId) {
+        Optional<Ticket> optionalTicket = Optional.ofNullable(ticketRepository.findByUserIdAndId(userId, ticketId));
+        Ticket ticket = optionalTicket.orElseThrow(() -> new TicketException(ErrorCode.FAIL_TO_FIND_TICKET));
```

Comment:
`TicketRepository`의 `findByUserIdAndId` 메서드의 리턴값을 `Optional<Ticket>`으로 받게 하면 여기서 `Optional`로 래핑할 필요 없을 것 같습니당 :)
```suggestion
        Ticket ticket = ticketRepository.findByUserIdAndId(userId, ticketId)
        .orElseThrow(() -> new TicketException(ErrorCode.FAIL_TO_FIND_TICKET));
```